src/plugins/intel_cpu/src/memory_desc/dnnl_blocked_memory_desc.h

```diff
@@ -11,6 +11,10 @@
 namespace ov {
 namespace intel_cpu {
 
+#if defined(_MSC_VER)
+#    pragma warning(push)
+#    pragma warning(disable : 4250)
```

Comment:
Whould be nice to put comment here why this warning is suppressed.

Reply:
And fix all warnings. This is Microsoft and SDL request.
Here is C4250 details: https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-2-c4250?view=msvc-170

Reply:
Yeah. I mentioned to put this comment into the code. Smt like: "Disabled MSVC warning about inheritance via dominance".
Otherwise it is just magic number

Reply:
Can we also have a wrapper for this in scope utils file?, i.e.:
```cpp
#define DISABLE_WARNING_MSVC_BEGIN(id) \
#if defined(_MSC_VER)
#    pragma warning(push)
#    pragma warning(disable : id)
#endif

#define DISABLE_WARNING_MSVC_END(id) \
#if defined(_MSC_VER)
#    pragma warning(pop)
#endif
...

DISABLE_WARNING_MSVC_BEGIN(4250)
...
DISABLE_WARNING_MSVC_END(4250)
```
Will look much more clean

Reply:
It's good idea, I did some update, please review it again.
Because functional test and gpu pulgin also use this it, so I warp this Macro: src/common/util/include/openvino/util/util.hpp, @EgorDuplensky

Reply:
Actually, I am not sure whether it is reasonable, @ilya-lavrenov @dmitry-gorokhov .
Or where to wrap?

Reply:
I think you can add it to this file: https://github.com/openvinotoolkit/openvino/blob/master/src/common/util/include/openvino/util/util.hpp